bundle/edu.gemini.lchquery.servlet/src/main/scala/edu/gemini/lchquery/servlet/LchQueryFunctor.scala

```diff
@@ -61,7 +61,7 @@ class LchQueryFunctor(queryType: LchQueryFunctor.QueryType,
 
     def makeTargetNode(target: SPTarget, env: TargetEnvironment): Option[Serializable] = {
       def targetType: Option[String] = {
-        if (env.isBasePosition(target)) Some("Base")
+        if (env.getArbitraryTargetFromAsterism eq target) Some("Base")
```

Comment:
All uses of `getArbitraryTargetFromAsterism` will be crushed so we will have a chance to revisit this.